flocker/node/script.py

```diff
@@ -66,6 +68,40 @@ def flocker_dataset_agent_main():
     agent_script = AgentScript(service_factory=service_factory.get_service)
     options = DatasetAgentOptions()
 
+    # Use CPU time instead of wallclock time.
+    # The control service does a lot of waiting and we do not
```

Comment:
I'm fine with either here or in a different PR. +1 to cleaning up. Will address before merging.

Please add all sorts of comments like this to this PR; most of my making of this PR was:

* I want to have tests running on all these changes.
* I want to have a single location for people to add TODOs before we merge this (as github comments).